server/test/com/cloud/network/firewall/FirewallManagerTest.java

```diff
@@ -52,9 +60,9 @@
 import com.cloud.network.rules.FirewallRuleVO;
 import com.cloud.utils.component.ComponentContext;
 
-@Ignore("Requires database to be set up")
-@RunWith(SpringJUnit4ClassRunner.class)
-@ContextConfiguration(locations = "classpath:/testContext.xml")
+//@Ignore("Requires database to be set up")
+@RunWith(MockitoJUnitRunner.class)
+//@ContextConfiguration(locations = "classpath:/testContext.xml")
```

Comment:
I don't know. As @remibergsma mentioned, we just got the thing running with 7 PRs.

It's very good to have you looking into it, @borisroman . I really mean it! :) It means that when those PRs were pushed by the original authors nobody looked at them at all!

Thanks!